Client/src/Pages/SetNewPassword/index.jsx

```diff
@@ -1,14 +1,69 @@
 import BackgroundPattern from "../../Components/BackgroundPattern/BackgroundPattern";
 import "./index.css";
-import React from "react";
 import LockIcon from "../../assets/Images/lock-icon.png";
 import PasswordTextField from "../../Components/TextFields/Password/PasswordTextField";
 import Check from "../../Components/Check/Check";
 import Button from "../../Components/Button";
 import LeftArrow from "../../assets/Images/arrow-left.png";
 import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { newPasswordValidation } from "../../Validation/validation";
+import axios from "axios";
+const BASE_URL = import.meta.env.VITE_APP_API_BASE_URL;
+import { useNavigate } from "react-router-dom";
 
 const SetNewPassword = () => {
+  const navigate = useNavigate();
+  const [errors, setErrors] = useState({});
+  const [form, setForm] = useState({});
+  const [isLoading, setIsLoading] = useState(false);
+
+  const idMap = {
+    "register-password-input": "password",
+    "confirm-password-input": "confirm",
+  };
+
+  const handleInput = (e) => {
+    const fieldName = idMap[e.target.id];
+    setForm({ ...form, [fieldName]: e.target.value });
+    console.log(errors);
+  };
+
+  const handleSubmit = async () => {
+    // TODO show loading spinner
+    setIsLoading(true);
+    try {
+      await axios.post(`${BASE_URL}/auth/recovery/validate`, {
+        recoveryToken: token,
+      });
+      await axios.post(`${BASE_URL}/auth/recovery/reset`, {
+        ...form,
+        recoveryToken: token,
+      });
+      navigate("/login");
```

Comment:
We are redirecting to the `/new-password-confirmed` route rather than the `/login` route.

Reply:
Done!